src/components/03-components/06-dashboard/16-tile/tile.config.yml

```diff
@@ -13,3 +13,12 @@ variants:
     context:
       label:
         classes: "sds-tile--outline"
+
```

Comment:
List and SDS Tile are identical on site/update file naming https://fractal.build/guide/components/creating-a-component.html#creating-variants